### PR TITLE
Add a switch to control virsh pre conn to source hypervisor

### PR DESCRIPTION
**Pull request by xiaodwan**
Add a new parameter 'skip_virsh_pre_conn' to control pre-connection
to source hypervisor.
By default, 'skip_virsh_pre_conn' is None, it means v2v will do pre-
connection to source hypervisor. In some testing envrionment, like
gating, It doesn't have source hypervisor to connect, you need to
set 'skip_virsh_pre_conn' to 'yes' to skip the connection.

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
skip_virsh_pre_conn = yes
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.vmx.vmx_default.latest7.output_mode.rhev.rhv: PASS (1953.04 s)
skip_virsh_pre_conn = no
(1/1) type_specific.io-github-autotest-libvirt.convert_from_file.positive_test.linux.input_mode.vmx.vmx_default.latest7.output_mode.rhev.rhv: PASS (1970.69 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh @kylazhang Please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -950,7 +951,11 @@ def _v2v_pre_cmd():
         create virsh instance, etc.
         """
         # Cannot get mac address in 'ova', 'libvirtxml', etc.
-        if input_mode not in ['disk', 'libvirtxml', 'local', 'ova']:
+        if input_mode not in [
+            'disk',
+            'libvirtxml',
+            'local',
+                'ova'] and not skip_virsh_pre_conn:
```
Comment (kylazhang) [inline virttest/utils_v2v.py]: Please fix the aligning issue

Comment (xiaodwan) [reply virttest/utils_v2v.py]: This is not an aligning issue. The format is created by autopep8. 
I think this line is intent to distinct with following line.

**Pull request merged**